src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Ball.Colors getRandomColor() {
```

Comment:
Avoid using static methods as per the checklist. Instead, instantiate the class where you need to use its methods.